inference-engine/src/transformations/include/ngraph_ops/nms_ie.hpp

```diff
@@ -35,5 +35,26 @@ class TRANSFORMATIONS_API NonMaxSuppressionIE : public Op {
     bool m_sort_result_descending = true;
 };
 
+class TRANSFORMATIONS_API NonMaxSuppressionIE2 : public op::Op {
```

Comment:
Could you please reflect difference from `NonMaxSuppressionIE` in name instead of just suffix `2`?

Reply:
To be honest, I don't know how to call it differently but not too long name. Also, we have a common approach that we use an opset version number to distinguish operation versions. So I assume this approach is fine here.